### PR TITLE
New cloud watch portal

### PR DESCRIPTION
**Pull request by chloegugli**

### CODE REVIEW

File: app/templates_versions/enterprise/subviews.json [hunk 1]
```diff
@@ -118,6 +118,19 @@
         },
         "images": ["not_available_image_cloud_automation.png"]
     },
+    "cloudWatch": {
```
Comment (ShatalovYaroslav) [inline app/templates_versions/enterprise/subviews.json]: Still I am not sure if a naming has impact here. Could you test this somehow? For example put not available and generate dist.

File: app/templates_versions/community/subviews.json [hunk 2]
```diff
@@ -56,6 +56,19 @@
             "services": ["JobPlannerRestService"]
         }
     },
```
Comment (ShatalovYaroslav) [inline app/templates_versions/community/subviews.json]: The same comment as below.

File: app/templates_versions/enterprise/subviews.json (hunk 1 above)
Comment (chloegugli) [reply app/templates_versions/enterprise/subviews.json]: This name is a JSON key, we could call it however we want

File: app/templates_versions/community/subviews.json (hunk 2 above)
Comment (chloegugli) [reply app/templates_versions/community/subviews.json]: This name is important as it should be the name of the Service we define on the job planner portal project. If we don't import it here, it won't work

File: app/templates_versions/enterprise/subviews.json (hunk 1 above)
Comment (ShatalovYaroslav) [reply app/templates_versions/enterprise/subviews.json]: Ok. I see. So we don't call this key name for now somewhere in project automatically? These names are used somewhere already?

File: app/templates_versions/community/subviews.json (hunk 2 above)
Comment (ShatalovYaroslav) [reply app/templates_versions/community/subviews.json]: Ok

File: app/templates_versions/enterprise/subviews.json (hunk 1 above)
Comment (chloegugli) [reply app/templates_versions/enterprise/subviews.json]: Yes, in grunt build as a key to get portals information

Comment (ShatalovYaroslav) [reply app/templates_versions/enterprise/subviews.json]: Ok. So the PR is approved.

**Pull request merged**